src/components/sales/OfflineOrderBill.vue

```diff
@@ -0,0 +1,422 @@
+				<p class="text-center">Sales Invoice</p>
+			</div>
+		</div>
+
+		<div id="date-section" class="mb-5 pt-5">
+			<p class="text-center">{{getCurrentTime()}}</p>
+			<p class="text-center">{{getCurrentDate()}}</p>
+		</div>
+
+
+		<div id="order-items-section" class="mb-5">
+			<table>
+				<colgroup>
+					<col span="1" style="width: 5%;">
+					<col span="1" style="width: 30%;">
+					<col span="1" style="width: 10%;">
+					<col span="1" style="width: 15%;">
+					<col span="1" style="width: 15%;">
+					<col span="1" style="width: 25%;">
+				</colgroup>
+
+				<tr>
+					<th>Sr#</th>
+					<th>Item's Name</th>
+					<th>Qty</th>
+					<th>Rate</th>
+					<th>Amount</th>
+				</tr>
+				<tr v-for="(orderItem, index) in order.order_item" v-bind:key="orderItem.id">
+					<td>{{ index+1 }}</td>
+					<td>{{product_name[index]}}</td>
+					<td>{{ trimNumber(orderItem.quantity) }}</td>
+					<td>{{ trimNumber(orderItem.price) }}</td>
+					<td>{{ getItemTotal(orderItem.price, orderItem.quantity) }}</td>
+				</tr>
+			</table>
+		</div>
+
+		<div id="totals-section" class="mb-5">
+			<div id="col-1">
+				<table style="font-size: 12px;">
+					<colgroup>
+						<col span="1" style="width: 75%;">
+						<col span="1" style="width: 25%;">
+					</colgroup>
+					<tr>
+						<td><strong>Total Units: </strong></td>
+						<td>{{order.order_item.length}}</td>
+					</tr>
+					<tr>
+						<td><strong>Discount: </strong></td>
+						<td>{{ trimNumber(order.total_discount) }}</td>
+					</tr>
+				</table>
+			</div>
+			<div id="col-2">
+				<table style="font-size: 12px;">
+					<colgroup>
+						<col span="1" style="width: 70%;">
+						<col span="1" style="width: 30%;">
+					</colgroup>
+					<tr>
+						<td><strong>Total: </strong></td>
+						<td>{{total}}</td>
+					</tr>
+					<tr>
+						<td><strong>Net Payable: </strong></td>
+						<td>{{trimNumber(order.total)}}</td>
+					</tr>
+					<tr>
+						<td><strong>Received: </strong></td>
+						<td>{{trimNumber(order.amount_received)}}</td>
+					</tr>
+					<tr>
+						<td><strong>Change: </strong></td>
+						<td>{{change}}</td>
+					</tr>
+				</table>
+			</div>
+		</div>
+
+		<div id="footer-section" style="width:80mm">
+			<p class="text-center" style="font-size: 14px;"><strong>Thanks for Visiting</strong></p>
+			<p class="text-center" style="font-size: 14px;"><strong>Powered by CoalDev</strong></p>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { OrderItem } from '@/store/models/orderItem';
+import { ActionTypes } from '@/store/modules/order/actions';
+import { defineComponent } from 'vue';
+import {mapActions, mapGetters } from 'vuex';
+import offlineStoreService from "@/utils/offline-store/index";
+
+export default defineComponent({
+  name: 'OfflineOrderBill',
+  data(){
+		return{
+			printDelay: 200,
+			order: offlineStoreService.getLatestOrder(),
+		}
+  },
+  methods:{
+		getElementTag: function(tag: keyof HTMLElementTagNameMap): string {
```

Comment:
@ayeshatasaduq  @khunshan-coaldev created separate module for printing any component. Use that here, you may coordinate with him.